VWOSdk.Tests/VWOSdk.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp2.0</TargetFramework>

```

Comment:
reason?

Reply:
bro, we had netcoreapp2.0 installed, isliye